### DISCUSSION TITLE
v0.8.7

### DISCUSSION THREAD

**Discussion by felddy** [felddy/foundryvtt-docker#219] [General]
**Stable** release of `0.8.7`

https://foundryvtt.com/releases/0.8.7

# ⚠️ WARNING # 

WARNING: Version `0.8.7` is labeled as a stable release, but there is always the possibility of unexpected bugs or module compatibility issues. We do not recommend updating immediately prior to a game session unless you are confident in your own ability to troubleshoot any issues that arise.

<hr /><em>This discussion was created from the release <a href='https://github.com/felddy/foundryvtt-docker/releases/tag/v0.8.7'>v0.8.7</a>.</em>

**Comment by jocolamarco** [comment]
Is the new A/V implementation with simplepeer working with the 0.8.7 docker release? I attempted this but neither myself nor my players could talk to - I'm wondering if this is related with any of the settings...

**Reply by felddy** [reply]
I haven't tested the [simple-peer](https://github.com/feross/simple-peer) implementation yet.  If it changed any of the port publishing requirements it could require some changes in your configuration.  I'll have to dig into the [documentation](https://foundryvtt.com/article/audio-video/) to see what configurations have changed.

Thanks for pointing this out.

**Reply by jocolamarco** [reply]
Just to let you know, the Jitsi module works perfectly in this version.
